spec/app.spec.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-env jest, browser */
+import App from '../src/javascripts/modules/app'
+import { CLIENT, ORGANIZATIONS } from './mocks/mock'
```

Comment:
WHY HAVE THESE CAPITALIZED?

Reply:
Why have the mocks in a seperate file? They seem only used in this spec, so why not bring them into this file?

Reply:
- BECAUSE THEY ARE CONSTANTS (quote: By convention, the constant identifiers in JavaScript are in uppercase.)
- This is more of providing a structure/skeleton as scaffold, mocks(e.g. CLIENT) will be reused, and can be much longer than the actual test code(mock data).

Reply:
I think calling it CLIENT is not very helpful for the developer. All throughout the code the variable is called `client` and suddenly in tests it becomes `CLIENT`.

Reply:
👍 makes sense, I can fix it.